### PR TITLE
No longer test against casatools and casatasks

### PR DESCRIPTION
**Pull request by astrofrog**
... since spectral-cube now depends on casa-formats-io by default

Fixes https://github.com/glue-viz/glue-astronomy/pull/65

### CODE REVIEW

**Comment by dhomeier** [comment]
Has the interface/format changed from casatools to casa-formats-io?
```
            if os.path.isdir(datafn):
>               raise OSError("Tried to access a data file that's actually "
E               OSError: Tried to access a data file that's actually a package data director
```
`cube_3d.image` does not look like any standard format, but I currently cannot access any pkg_data  files under `data/` (getting a 404 from two different IPs here), so cannot check.

**Comment by dhomeier** [comment]
> if os.path.isdir(datafn):
> >               raise OSError("Tried to access a data file that's actually "
> E               OSError: Tried to access a data file that's actually a package data directory

Ah, it's `get_pkg_data_filename` itself stumbling on this:
https://github.com/glue-viz/glue-astronomy/pull/64/files#r814350245
@pllim is there any trick to work around this using, say `get_pkg_data_filenames` (plural!) or `get_pkg_data_path`? Both seem to be able to retrieve the local path in `tests/data`, but I suppose they will not take care of the automatic remote retrieval?

File: setup.cfg [hunk 1]
```diff
@@ -13,15 +13,15 @@ long_description_content_type = text/x-rst
...
+python_requires = >=3.7
 setup_requires =
     setuptools_scm
 install_requires =
     astropy>=4.0
     glue-core>=1.0
     regions>=0.4
     specutils>=0.7
-    spectral-cube>=0.5.0
+    spectral-cube>=0.6.0
```
Comment (pllim) [inline setup.cfg:24]: I still don't like this. See #62

**Comment by pllim** [comment]
@astrofrog , @dhomeier , see astrofrog/glue-astronomy#1 (PR to this PR)

**Pull request merged**